src/style.scss

```diff
@@ -0,0 +1,105 @@
+body {
+  background-color: lightgray;
+}
+
+h1 {
+  text-align: center;
+}
+
+img {
+  display: block;
+  margin: 0 auto;
+  width: 400px;
+  max-width: 100%;
+}
+
+p {
+  margin: 0 auto;
+  text-align: center;
+  width: 80%;
+}
+
+.input-radio {
+  display: none;
+}
+
+.tab-label {
+  cursor: pointer;
+  transition: all 0.4s ease;
+  padding: 1em 2em;
+  max-width: 33%;
+  &:hover {
+    background-color: cornflowerblue;
+    color: white;
+  }
+}
+
+.input-radio:checked + .tab-label {
+  background-color: cornflowerblue;
+  color: white;
+}
+
+/*
+.tabs {
+  display: flex;
+  position: relative;
+}
+
+.tabs::before,
+.tabs::after {
+  content: "";
+  display: table;
+}
+
+.tabs::after {
+  clear: both;
+}
+
+.tab-label {
+  position: relative;
+  text-align: center;
+  display: block;
+  line-height: 2.75em;
+  height: 3em;
+  width: fit-content;
+  padding: 0 20px;
+  top: 0;
+  cursor: pointer;
+  transition: all 0.25s ease;
+  &:hover {
+    background-color: cadetblue;
+    color: white;
+    top: -0.25rem;
+    transition: top 0.25s ease;
+  }
+}
+
+.tab-content {
+  position: absolute;
+  opacity: 0;
+  left: 0;
+  top: 3em;
+}
+
+.tab-switch {
+  display: none;
+}
+
+.tab-switch:checked + .tab-label{
+  background-color: cadetblue;
+  transition: all 0.4s;
+  color: white;
+  top: -0.0625rem;
+}
+
+.tab-switch:checked + label + .tab-content {
+  z-index: 2;
+  opacity: 1;
+  transition: all 0.35s;
+}
+*/
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))